src/parser/differential-rewards.ts

```
@@ -0,0 +1,155 @@
+import { Context } from "@ubiquity-os/permit-generation";
+import { createClient } from "@supabase/supabase-js";
+import type { Database } from "../adapters/supabase/types/database";
+import { Beneficiary, Result, ResultEntry } from "../types/results";
```

ESLint

[error] 'Beneficiary' is defined but never used. Allowed unused vars must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] Remove this unused import of 'Beneficiary'.

(sonarjs/unused-import)